.circleci/config.yml

```diff
@@ -92,7 +92,7 @@ jobs:
             sudo apt-get install -y python-rosdep
             cd ~/project/robots/LoCoBot/install
             chmod +x locobot_install_all.sh
-            ./locobot_install_all.sh -t sim_only -p 2
+            ./locobot_install_all.sh -t sim_only -p 2 -l interbotix
```

Comment:
Just wanted to make sure the logic in the installation script worked regardless of which robot was chosen (the distro chosen for each robot was completely arbitrary). On a different note, it might be a good idea to add another test case where you install the full version of the locobot so you can test that librealsense and the associated realsense-ros package installs successfully. The only caveat would be how to handle running the gen_cfg.py script at the end as that can't be done without having a physical camera connected...

On a different note, I just wanted to point out that the installation script pulls from the Develop branch currently. This will have to be updated to 'master' if and when it's ready to be merged.

Reply:
> On a different note, I just wanted to point out that the installation script pulls from the Develop branch currently. This will have to be updated to 'master' if and when it's ready to be merged.

Yes, I will make a note of that.